tsconfig.json

```
@@ -3,5 +3,11 @@
   "references": [
     { "path": "./tsconfig.app.json" },
     { "path": "./tsconfig.node.json" }
-  ]
+  ],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
```

Ao invés de definir a rota base e navegar a partir dela poderia definir para cada camada para deixar mais bonitinho:
ex:
```suggestion
      "@atomic*": ["./src/atomic"]
       "@atomic/*": ["./src/atomic/*"]
```